reana_workflow_controller/workflow_run_managers/kubernetes/__init__.py

```diff
@@ -0,0 +1,90 @@
+# This file is part of REANA.
+# Copyright (C) 2018 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Workflow run manager for Kubernetes backends."""
+
+from kubernetes import client
+
+from reana_commons.k8s.api_client import current_k8s_batchv1_api_client
+from reana_workflow_controller.config import (MANILA_CEPHFS_PVC,
+                                              SHARED_FS_MAPPING,
+                                              REANA_STORAGE_BACKEND)
+from reana_workflow_controller.workflow_run_managers import WorkflowRunManager
+
+
+class KubernetesWorkflowRunManager(WorkflowRunManager):
+    """Implementation of WorkflowRunManager for Kubernetes."""
+
+    k8s_shared_volume = {
+        "ceph": {
+            "name": "default-shared-volume",
+            "persistentVolumeClaim": {
+                "claimName": MANILA_CEPHFS_PVC,
+                "readOnly": False,
+            }
+        },
+        "local": {
+            "name": "default-shared-volume",
+            "hostPath": {
+                "path": SHARED_FS_MAPPING['MOUNT_SOURCE_PATH'],
+            }
+        }
+    }
+    """Configuration to connect to the different storage backends."""
+
+    def start_batch_workflow_run(self):
+        """Start a batch workflow run."""
+        namespace = "default"
```

Comment:
I think here we should have single quotes

Reply:
Regarding this, there is already a mix in the code base what do you think about using something like https://github.com/ambv/black? @reanahub/developers

Reply:
"Black prefers double quotes over single quotes" :wink: